docker/Dockerfile.worker

```diff
@@ -66,7 +66,7 @@ WORKDIR /workspace
 
 COPY go.mod go.sum ./
 RUN go mod download
-COPY . .
+COPY --exclude=sdk . .
```

Comment:
I don't think this is needed here since the SDK is being copied into `/workspace` [here](https://github.com/beam-cloud/beta9/pull/229/files#diff-d0259383c9168a0ceb522864a6877e2ad094e9661fc4b86821763fe8beffeeceR112).

Doing this allows for quicker changes to the SDK in the worker when go code hasn't been modified.